packages/types/app/index.d.ts

```diff
@@ -15,6 +15,7 @@ type NuxtState = Record<string, any>
 export interface Context {
   app: NuxtAppOptions
   base: string
+  $config: Record<string, any>
```

Comment:
Maybe sort in a real alphabetical manner (`$` before alpha chars)?

Reply:
@pi0 What's your POV re this ?

Reply:
IMO it's better as sorting it this way is kinda arbitrary.
And you can't sort it automatically with "sort" function in your editor.

Reply:
Makes sense :)